src/celengine/star.h

```diff
@@ -124,7 +141,7 @@ class StarDetails
 
     std::string infoURL;
 
-    std::unique_ptr<std::vector<Star*>> orbitingStars{ nullptr };
+    std::vector<Star*> orbitingStars{ };
```

Comment:
std::vector is 32 bytes while pointer is 8 bytes only (in 64 bit systems). i don't think it's an equivalent replacement